### DISCUSSION TITLE
DB password written to wp-config.php automatically?

### DISCUSSION THREAD

**Discussion by MackSix** [leokhoa/laragon#400] [General]
Laragon will create a WordPress site, but makes a blank database password in the wp-config.php file, so I have to manual put the password in before the site will install. It would be nice for Laragon to write a password to wp-config.php.

**Comment by rescenic** [comment]
Just download wordpress.zip, place the extracted folder in "www" folder. Start apache & mysql, type in browser: localhost/wordpress

**Reply by MackSix** [reply]
I know how to manually install WordPress, that is irrelvant to my request. It also takes longer than me using quickapp and then editing the wp-config.php file to put my database password in.

**Comment by ajaxthemestudios** [comment]
Facing the same problem. It was not so before. Once you finish installation and try to open the page, it brings - **Error establishing a database connection**
When you check the wp-config file, it shows that the password to the database was not added resulting into the error.
It can be done manually but it is not ideal for a quick tool.
